src/lib/ngsi/ParseData.h

```diff
@@ -253,14 +263,15 @@ typedef struct UpdateContextData
 *
 * UpdateContextResponseData - 
 */
-typedef struct UpdateContextResponseData
+struct UpdateContextResponseData
 {
+  UpdateContextResponseData(): cerP(0), attributeP(0), metadataP(0), domainMetadataP(0) {}
```

Comment:
Not that is changes anything (and it yields longer lines), but using NULL instead of 0 might be clearer.
I'm not sure what I prefer even myself ...